app/_how-tos/run-docker-images.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Run Docker images
```

Comment:
```suggestion
title: Run {{site.base_gateway}} from a custom Docker image
```
I don't know Docker phrasing that well, but I wanted to make this clear that the user would be getting Kong Gateway in the end, and it should be "custom" image I think.